sdk/python/kfp/dsl/types.py

```diff
@@ -12,6 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
```

Comment:
I don't think that's your fault but the indentation of this file does not look right.

Reply:
Do you mean the double blank lines? What would you prefer?

Reply:
No. I mean the indentation is made by tab instead of by space, which makes it look much bigger than other files.

Reply:
Fixed.